test/gtest/ucp/test_ucp_proto_mock.cc

```diff
@@ -871,6 +871,8 @@ UCS_TEST_P(test_ucp_proto_mock_self, rkey_ptr)
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_proto_mock_self, self, "self")
 
+#ifdef HAVE_CUDA
```

Comment:
can you pls also fix initialization for test_ucp_proto_mock_gpu test on line 918 and remove rocm?
It was not added in this PR, but that test expects cuda ppln protocols to be selected anyway and we make this test available only for cuda transport